interp/build.go

```diff
@@ -86,9 +87,29 @@ func buildTagOk(s string) (r bool) {
 }
 
 // goNumVersion returns the go minor version number
-func goNumVersion() int {
-	v := strings.Split(runtime.Version(), ".")
-	n, _ := strconv.Atoi(v[1])
+func goNumVersion(version string) int {
```

Comment:
any reason why we don't use `go/build.Context` instead of `runtime.Version`?
- https://godoc.org/go/build#Context

ie: a combination of `build.Default.ReleaseTags` and `build.Default.BuildTags` ?

with my current dev environment:
```
$> go version
go version devel +60f14fddfe Wed Jun 26 16:35:14 2019 +0000 linux/amd64
```

the following program:
```go
package main

import (
	"go/build"
	"log"
)

func main() {
	ctx := build.Default
	log.Printf("build-tags:   %v", ctx.BuildTags)
	log.Printf("release-tags: %v", ctx.ReleaseTags)
}
```

prints:
```
$> go run ./main.go
2019/07/30 10:38:50 build-tags:   []
2019/07/30 10:38:50 release-tags: [go1.1 go1.2 go1.3 go1.4 go1.5 go1.6 go1.7 go1.8 go1.9 go1.10 go1.11 go1.12 go1.13]
```

Reply:
The `build.Default.ReleaseTags` simplify the version parsing for devel/beta/rc.

`build.Default.BuildTags` seems less relevant for handling version.
 
> BuildTags is not set for the Default build Context.

It can be useful for handling custom build tags, for that we could to add a `build.Context` to the `buildOk` method and allow to define custom build context as an option of the interpreter.

The purpose of this PR is to manage the version, then I will do another PR after the merge of this PR to manage that.